pages/common/go-env.md

```diff
@@ -0,0 +1,16 @@
+# go env
+
+> Tool for printing Go environment information.
```

Comment:
I know this is what the documentation says. But the offocial docs aim to be very canonical and terse. This can probably be re-worded as "Manage environment variables used by the Go toolchain".